care/facility/summarisation/facility_capacity.py

```diff
@@ -1,47 +1,71 @@
 from datetime import timedelta
 
 from celery.decorators import periodic_task
+from celery.schedules import crontab
 from django.utils.decorators import method_decorator
 from django.views.decorators.cache import cache_page
-from rest_framework import status, viewsets
+from django.views.decorators.vary import vary_on_cookie
+from rest_framework import serializers, status, viewsets
+from rest_framework.generics import get_object_or_404
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin
```

Comment:
Codacy found an issue: [Unable to import 'rest_framework.mixins'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5811718)